packages/zero-next-plugin/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@mui/zero-next-plugin",
+  "version": "0.0.1-alpha.3",
```

Comment:
@brijeshb42 this is a wrong version, we just created the package. Was it intentional so that it is the same version as the other zero-related packages? In my opinion they should anyway be all at version `0.0.1-alpha.3` until we release them publicly.

Reply:
Yes. It was set to match other zero package versions.